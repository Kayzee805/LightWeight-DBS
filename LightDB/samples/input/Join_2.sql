SELECT * FROM Professors, Courses WHERE Professors.id = Courses.professor_id
