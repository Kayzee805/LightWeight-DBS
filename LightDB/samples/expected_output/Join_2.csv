201,41,304,11,201,30
202,46,301,7,202,50
202,46,303,8,202,125
203,52,302,9,203,40
203,52,305,10,203,35
